src/app/core/shell/sidenav/sidenav.component.scss

```diff
@@ -134,8 +134,10 @@
   .app-brand {
     .app-logo {
       margin: auto;
+      &:hover {
+        cursor: pointer;
+      }
     }
-
     .app-logo-text {
```

Comment:
dont remove this line

Reply:
> dont remove this line (138)

Left 141 as a blank line.

Reply:
Please review if the required change has been done properly